usaspending_api/submissions/models/dabs_submission_window_schedule.py

```diff
@@ -23,3 +24,25 @@ class Meta:
             "submission_fiscal_month",
             "is_quarter",
         )
+
+    @classmethod
+    def latest_monthly_to_display(cls):
+        return cls.get_latest_submission(is_quarter=False)
+
+    @classmethod
+    def latest_quarterly_to_display(cls):
+        return cls.get_latest_submission(is_quarter=True)
+
+    @classmethod
+    def get_latest_submission(cls, is_quarter):
+        return (
+            cls.objects.filter(is_quarter=is_quarter, submission_reveal_date__lte=datetime.now(timezone.utc))
+            .order_by("-submission_fiscal_year", "-submission_fiscal_quarter", "-submission_fiscal_month")
+            .values(
+                "submission_fiscal_year",
+                "submission_fiscal_quarter",
+                "submission_fiscal_month",
+                "submission_reveal_date",
+            )
+            .first()
+        )
```

Comment:
Should these return the whole DABSSubmissionWindowSchedule object?  You only need `submission_fiscal_year`, `submission_fiscal_quarter`, `submission_fiscal_month`, and `submission_reveal_date`, but if we want to prevent clones of this function popping up all over the place, we should probably return all of the fields/columns.

Reply:
Done